collector/Makefile

```
@@ -25,13 +28,19 @@ cmake-build/collector: $(HDRS) $(SRCS) txt-files $(shell find falcosecurity-libs
 		-e DISABLE_PROFILING="true" \
 		quay.io/stackrox-io/collector-builder:$(COLLECTOR_BUILDER_TAG) /build-collector.sh
 
-container/bin/collector: cmake-build/collector
+.PHONY: post-build
+post-build:
 	mkdir -p container/bin
 	mkdir -p container/libs
 	cp "$(COLLECTOR_BIN_DIR)/collector" container/bin/collector
 	cp -r "$(CMAKE_DIR)"/THIRD_PARTY_NOTICES/* container/THIRD_PARTY_NOTICES/
 	cp "$(LIBSINSP_BIN_DIR)/libsinsp-wrapper.so" container/libs/libsinsp-wrapper.so
 
+container/bin/collector: 
+	make pre-build
+	make cmake-build/collector 
+	make post-build
```

```suggestion
	$(MAKE) pre-build
	$(MAKE) cmake-build/collector 
	$(MAKE) post-build
```
I believe that this is closer to the canonical way